Documentation/troubleshooting.rst

```diff
@@ -156,11 +156,11 @@ little bit different
 
 .. Note::
 
-          Please check the dump files for sensitive informations and strip it
+          Please check the dump files for sensitive information and strip it
           away before sharing it with us.
 
-Below is a approximate list of what kind of information is in the archive, but
-you should still verify before sharing.
+Below is an approximate list of the kind of information in the archive. It is
+ recommeded that you verify it before sharing.
 
```

Comment:
Remove the space before `recommeded`, otherwise part of the sentence will be bold.

Reply:
Done.